python/rds/app.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+from aws_cdk import (
+    aws_ec2 as ec2,
+    aws_rds as rds,
+    core
```

Comment:
I could see a trailing comma here as well, although it's not mandatory (`core` is usually the last import).